lib/folio/mhld_builder.rb

```diff
@@ -84,7 +84,7 @@ def latest_received(holding_id)
       return unless latest_piece && order_is_ongoing_and_open?(latest_piece)
 
       enumeration = latest_piece['enumeration'] # may not be present
-      chronology = latest_piece.fetch('chronology')
+      chronology = latest_piece['chronology'] # may not be present
       enumeration ? "#{enumeration} (#{chronology})" : chronology
```

Comment:
Do we need to tweak the string output in that case?

```suggestion
      enumeration && chronology ? "#{enumeration} (#{chronology})" : enumeration || chronology
```

Reply:
It currently looks to have parity with what is coming out of Symphony, so I see no reason to make adjustments there.

Reply:
I'm surprised we'd leave a blank `()` sitting around if there was an enumeration but not a chronology, but it's difficult to  deciphering the existing logic and/or translate the MHLD fields to FOLIO data.

Is this the equivalent code?
https://github.com/sul-dlss/searchworks_traject_indexer/blob/main/lib/mhld_field.rb#L57-L62

Reply:
I don't know. This was just to fix the missing method error. If you think there's an error in the output, I would prefer that we make that a new issue, find an example record and add a test to show the behavior we want.

Reply:
🤷‍♂️